src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -2,7 +2,21 @@
 
 public class WorkWithFile {
 
```

Comment:
[Please don’t add redundant empty lines to your code.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#please-dont-add-redundant-empty-lines-to-your-code)

Reply:
check redundant lines in all classes